src/views/tutorial-view/server.ts

```diff
@@ -52,14 +52,14 @@ export async function getTutorialPageProps(
   const layoutProps = {
     headings,
     breadcrumbLinks: getTutorialsBreadcrumb({
-      product: { name: product.name, slug: product.slug },
+      product: { name: product.name, filename: product.slug },
```

Comment:
Just a rename here to make it more semantic 👍

Reply:
That's fair, I just wanted the interface to be consistent. If its a blocker I can change it, but it seems pretty minor to me 🤷‍♀️